redcap_bridge/server_elab_interface.py

```diff
@@ -0,0 +1,120 @@
+import json
+import os
+import elabapi_python
+
+from redcap_bridge.utils import conversion_csv_to_json
+
+
+def download_experiment(server_config_json, experiment_id):
+    api_client = get_elab_config(server_config_json)
+    experiment_api = elabapi_python.ExperimentsApi(api_client)
+
+    exp = experiment_api.get_experiment_with_http_info(experiment_id)
+    status_code = exp[1]
+
+    return exp, status_code
+
+
+def upload_template(server_config_json, template_file):
+    """
+    Create a template with metadata.
```

Comment:
```suggestion
    Upload a template to the server.
    
    If the template contains metadata these will be also automatically uploaded
```

Reply:
I have change all the comment and replace create by upload